client/src/components/common/Input.tsx

```diff
@@ -45,22 +49,32 @@ export default function Input({
     setIsVisible(!isVisible);
   };
 
-  function renderLabel() {
-    if (label) {
+  function renderLabelAndInfo() {
+    if (label || fieldInfo) {
       return (
-        <label
-          htmlFor={id}
-          className='block text-sm font-medium leading-6 text-gray-900'
-        >
-          {label}
-        </label>
+        //If No label, then fieldInfo will be to the end
+        <div className='flex items-center'>
+          {label && (
+            <label
+              htmlFor={id}
+              className='flex-1 text-sm font-medium leading-6 text-gray-900'
+            >
+              {label}
+            </label>
+          )}
+          {fieldInfo && (
+            <TooltipIcon tooltipText={fieldInfo} tooltipPosition={'left'}>
+              <InfoIcon width={20} height={20} />
```

Comment:
Un detalle pero capas estaria bueno pasarle `width` y `height` 13 que queda mas proporcional al label.
![Screenshot 2023-09-23 at 12 29 57](https://github.com/wyeworks/finder/assets/79933970/e79c4265-b84c-45dc-9a91-2332b5e62e56)

```suggestion
              <InfoIcon width={13} height={13} />
```

Reply:
Tenes razon, queda mas elegante